cli/flags.rs

```diff
@@ -108,6 +174,9 @@ pub struct Flags {
   pub allow_write: Option<Vec<PathBuf>>,
   pub cache_blocklist: Vec<String>,
   pub ca_file: Option<String>,
```

Comment:
Why keep this if we have ca_pem? I would prefer to have just one of the two.

Reply:
We'd have to read the file in flag parsing, and it wouldn't correspond well to the actual CLI flags. ~`ca_pem`~ `ca_data` is the one that doesn't belong here, but it's a particularly convenient hack.